### DISCUSSION TITLE
[Feature Idea] Create a tailwind.config.js file based on css

### DISCUSSION THREAD

**Discussion by mnaderian** [tailwindlabs/tailwindcss#2555] [Ideas]
## Idea Description  
Possibility to create a tailwind.config.js file out of a css file.

## Reasons
There are situations where someone might get one or a bunch of css files and wants to use the custom colors and some other elements and organize them in a tailwind.config.js file.
This can help develop the html template with tailwind css or can help a developer to migrate to TailwindCSS easily. 
There can be a possibility to convert from Bootstrap to TailwindCSS.

**Comment by tombohub** [comment]
So you mean for the tailwind to pick up all the colors from CSS file? Would that be the only colors or you would like to have whole palette with shades?

**Reply by m-naderian** [reply]
I mean we should have an option to be able to do both.

**Reply by tombohub** [reply]
ok, maybe I'll incorporate in my project: https://github.com/tombohub/palettolithic
https://palettolithic.com

**Comment by simonswiss** [comment]
I believe that what [cssstats](https://cssstats.com/) (code is open source) is doing would be a good inspiration to get started on this idea!
